src/utils/check-pr/cleanup_ad.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+import argparse
+from azure.common.client_factory import get_client_from_cli_profile
+from azure.graphrbac import GraphRbacManagementClient
```

Comment:
you should not be using graphrabac, it is being [deprecated](https://github.com/microsoft/onefuzz/issues/870)

Reply:
you need to use the microsoft graph rest api to [get](https://docs.microsoft.com/en-us/graph/api/application-get?view=graph-rest-1.0&tabs=http)and [delete](https://docs.microsoft.com/en-us/graph/api/application-delete?view=graph-rest-1.0&tabs=http). you can check the usage of `query_microsoft_graph` in registration.py for example on how to do it